packages/e2e-test-utils/src/deleteWidgets.js

```diff
@@ -18,8 +18,13 @@
  * Internal dependencies
  */
 import visitAdminPage from './visitAdminPage';
+import activatePlugin from './activatePlugin';
+import deactivatePlugin from './deactivatePlugin';
 
-async function deleteWidgets() {
+async function deleteWidgets(enableClassic = false) {
+  if (enableClassic) {
```

Comment:
Is there ever a case where we'd want to call `deleteWidgets` with `false`?

This function does not work without the classic widgets editor, so this flag seems a bit redundant.

Reply:
Yeah, it is false here - https://github.com/google/web-stories-wp/blob/d2a6f7f1ef86a0a160ef3ea9243021975e52ce20/packages/e2e-tests/src/specs/wordpress/widget.js#L31

This stop classic widgets being enabled twice. 

I added the param for a reason...

Reply:
> This stop classic widgets being enabled twice.

That's not an issue though, `activatePlugin` is a no-op if the plugin is already enabled.

Reply:
Removed.